dPhysics/dpWorld.cpp

```diff
@@ -10,14 +10,17 @@
 #include "dConfig.h"
 
 void dpWorld::Initialize(unsigned int zoneID, bool generateNewNavMesh) {
-	phys_sp_tilecount = std::atoi(Game::config->GetValue("phys_sp_tilecount").c_str());
-	phys_sp_tilesize = std::atoi(Game::config->GetValue("phys_sp_tilesize").c_str());
+	const auto physSpTilecount = Game::config->GetValue("phys_sp_tilecount");
```

Comment:
as just an fyi, auto cannot discern between reference and value and will always create a copy.  even if you specify const it will be a const T unless you specify const auto&